semgrep/semgrep/semgrep_main.py

```diff
@@ -176,6 +180,19 @@ def main(
         notify_user_of_work(filtered_rules, include, exclude)
 
     respect_git_ignore = not no_git_ignore
+
+    workdir = Path.cwd()
+    ignore_rules_file = f"{workdir}/{IGNORE_FILE_NAME}"
+    if os.path.exists(ignore_rules_file):
+        with open(ignore_rules_file) as f:
+            ignore_patterns = Parser(workdir).parse(f)
```

Comment:
Include debug logging for this; see [`get_semgrepignore`](https://github.com/returntocorp/semgrep-action/blob/dfa369e282ff93667611e3abc0b129a1d3aa16d1/src/semgrep_agent/semgrep.py#L122)